src/cascadia/WindowsTerminal/NonClientIslandWindow.cpp

```diff
@@ -1048,21 +1048,9 @@ void NonClientIslandWindow::_UpdateFrameMargins() const noexcept
 // - This method is called when the window receives the WM_NCCREATE message.
 // Return Value:
 // - The value returned from the window proc.
-[[nodiscard]] LRESULT NonClientIslandWindow::_OnNcCreate(WPARAM wParam, LPARAM lParam) noexcept
+[[nodiscard]] LRESULT NonClientIslandWindow::OnNcCreate(WPARAM wParam, LPARAM lParam) noexcept
 {
-    const auto ret = IslandWindow::_OnNcCreate(wParam, lParam);
-    if (!ret)
-    {
-        return FALSE;
-    }
-
-    // This is a hack to make the window borders dark instead of light.
-    // It must be done before WM_NCPAINT so that the borders are rendered with
-    // the correct theme.
-    // For more information, see GH#6620.
-    LOG_IF_FAILED(TerminalTrySetDarkTheme(_window.get(), true));
-
-    return TRUE;
+    return IslandWindow::OnNcCreate(wParam, lParam);

```

Comment:
Do we need this function at all now? We can just automatically roll up to the parent or use `using IslandWindow::OnNcCreate;`, right?

Reply:
Oh good point

Reply:
I know it's a nitpick but we still still don't need this function at all, right? We can just delete it and inherit it straight from IslandWindow?

Reply:
Oh gods I'm going blind. Only after drilling in to this comment on the `/files` page did I realize it wasn't on the `IslandWindow` version of the method. 

I gotta go get my eyes checked 😬